frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseFastForwardMergeToParentAction.java

```diff
@@ -22,7 +22,7 @@
 import com.virtuslab.logger.IEnhancedLambdaLogger;
 
```

Comment:
Pls also fix README.md (e.g. line 141) and features.md, just the text, no need to re-record the gifs yet IMO